### PR TITLE
[fix] Print actual config file path in wizard

### LINKED ISSUE #870: Config wizard always says it's editing the global config file
It's hard-coded... https://github.com/repobee/repobee/blob/8b0bcd2d0309ef97e526de8c551b6b9b85e89f0b/src/_repobee/ext/defaults/configwizard.py#L42-L49

Whereas it should check if the input `config` exists, and then say it's editing that.

### PR DESCRIPTION
**Pull request by MasterOdin**
Closes #870

This fixes the output for `repobee config wizard` so that it will use the actual config file path in the initial print of the file that will be overwritten, instead of the constant value.

### CODE REVIEW

File: tests/new_integration_tests/test_config_category.py [hunk 1]
```diff
@@ -139,7 +164,6 @@ def test_end_message_respects_config_file_argument(
             )
 
         # assert
-        assert (
-            f"Configuration file written to {config_file}"
-            in capsys.readouterr().out
+        assert capsys.readouterr().out.endswith(
+            f"Configuration file written to {config_file}\n"
```
Comment (slarse) [inline tests/new_integration_tests/test_config_category.py:168]: This change is unrelated to your patch. Even though it looks fine, I'm hesitant to merge something that changes production code _and_ a seemingly unrelated test.

Would you mind reverting this change?

**Review by slarse — CHANGES_REQUESTED** [review]
Looks great, thanks for taking the time! Could you just have a look at my one comment on the unrelated test change?

File: tests/new_integration_tests/test_config_category.py (hunk 1 above)
Comment (MasterOdin) [reply tests/new_integration_tests/test_config_category.py]: Done.

I can open a new PR with just this change?

Comment (slarse) [reply tests/new_integration_tests/test_config_category.py]: Sure!

**Comment by slarse** [comment]
@MasterOdin Thanks for your contribution!
